langchain/package.json

```diff
@@ -785,7 +792,9 @@
     "@elastic/elasticsearch": "^8.4.0",
     "@getmetal/metal-sdk": "*",
     "@getzep/zep-js": "^0.7.0",
-    "@gomomento/sdk": "^1.23.0",
+    "@gomomento/sdk": "^1.41.2",
+    "@gomomento/sdk-core": "^1.41.2",
+    "@gomomento/sdk-web": "^1.41.2",
```

Comment:
Sorry for not spotting this earlier, but we don't actually need `@gomomento/sdk-web` as a peer dep do we?

Reply:
Got it, no worries then.